purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -620,31 +631,26 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
     fun getCustomerInfo(
         callback: ReceiveCustomerInfoCallback
     ) {
-        retrieveCustomerInfo(identityManager.currentAppUserID, callback)
+        getCustomerInfo(CacheFetchPolicy.default(), callback)
     }
 
-    private fun retrieveCustomerInfo(
-        appUserID: String,
-        callback: ReceiveCustomerInfoCallback? = null
+    /**
+     * Get latest available purchaser info.
+     * @param fetchPolicy Specifies cache behavior for customer info retrieval
+     * @param callback A listener called when purchaser info is available and not stale.
+     * Purchaser info can be null if an error occurred.
+     */
+    fun getCustomerInfo(
+        fetchPolicy: CacheFetchPolicy,
+        callback: ReceiveCustomerInfoCallback
```

Comment:
We have Kotlin functions that accept functions instead of the callbacks in https://github.com/RevenueCat/purchases-android/blob/main/purchases/src/main/kotlin/com/revenuecat/purchases/listenerConversions.kt#L248, we should add one with the `fetchPolicy` argument